src/applications/financial-status-report/tests/unit/transform.unit.spec.jsx

```diff
@@ -0,0 +1,658 @@
+        },
+      },
+      {
+        fileNumber: '796121200',
+        payeeNumber: '00',
+        personEntitled: 'AJOHNS',
+        deductionCode: '71',
+        benefitType: 'CH33 Books, Supplies/MISC EDU',
+        diaryCode: '101',
+        diaryCodeDescription: 'Pending automatic benefit offset',
+        amountOverpaid: 0,
+        amountWithheld: 0,
+        originalAr: 166.67,
+        currentAr: 120.4,
+        debtHistory: [
+          {
+            date: '09/18/2012',
+            letterCode: '100',
+            description:
+              'First Demand Letter - Inactive Benefits - Due Process',
+          },
+        ],
+        id: 2,
+        resolution: {
+          resolutionType: 'Waiver',
+          agreeToWaiver: true,
+        },
+      },
+    ],
+    debt: {
+      currentAr: 0,
+      debtHistory: [
+        {
+          date: '',
+        },
+      ],
+      deductionCode: '',
+      originalAr: 0,
+    },
+    currEmployment: [
+      {
+        veteranGrossSalary: '5000.54',
+        deductions: [
+          {
+            name: 'Federal tax',
+            amount: '350.56',
+          },
+          {
+            name: 'State tax',
+            amount: '230.45',
+          },
+          {
+            name: 'Pension',
+            amount: '100',
+          },
+          {
+            name: 'Health insurance',
+            amount: '254.45',
+          },
+        ],
+        type: 'Full time',
+        from: '2017-1-XX',
+        to: '',
+        isCurrent: true,
+        employerName: 'Veteran Current One',
+      },
+      {
+        veteranGrossSalary: '2000.56',
+        deductions: [
+          {
+            name: 'Federal tax',
+            amount: '200.02',
+          },
+          {
+            name: 'Allotment',
+            amount: '134.56',
+          },
+          {
+            name: 'FICA (Social Security and Medicare)',
+            amount: '122.4',
+          },
+        ],
+        type: 'Part time',
+        from: '2019-7-XX',
+        to: '',
+        isCurrent: true,
+        employerName: 'Veteran Current Two',
+      },
+    ],
+    spCurrEmployment: [
+      {
+        spouseGrossSalary: '4000.45',
+        deductions: [
+          {
+            name: 'Federal tax',
+            amount: '325.54',
+          },
+          {
+            name: 'Health insurance',
+            amount: '250.34',
+          },
+          {
+            name: 'Garnishment',
+            amount: '150',
+          },
+        ],
+        type: 'Full time',
+        from: '2018-6-XX',
+        to: '',
+        isCurrent: true,
+        employerName: 'Spouse Current',
+      },
+    ],
+    financialOverview: {},
+    availableOptions: {},
+    income: [
+      {
+        veteranOrSpouse: 'VETERAN',
+        compensationAndPension: '3261.1',
+        education: '50',
+      },
+    ],
+    AGREED: true,
+  },
+};
+describe('fsr transform helper functions', () => {
+  describe('sumValue helper', () => {
```

Comment:
A good guiding princple is to have a test per conditional branch. `sumValue` (and `dateFormatter` ) have some `if`s. create a few tests to cover those branches in logic.